test/data-ingest/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
Why did you need to do a separate Dockerfile ?

Reply:
Mainly the Python confluent-kafka module seemed to assume specific hostnames? I can try without a Dockerfile again and see if I can fix the errors that happened.

Reply:
```
%3|1688045078.874|FAIL|rdkafka#producer-1| [thrd:kafka:9092/1]: kafka:9092/1: Failed to resolve 'kafka:9092': Temporary failure in name resolution (after 24ms in state CONNECT)
```

Reply:
oh so without a dedicated container your python code is running on the host and is thus unable to contact kafka? 

You can consider starting a ci-builder container with an entrypoint of `sleep infinity`  or `c.up(persistent=true)` . Then you can use `run` to run your script inside it, maybe with bin/pyactivate? Since the ci-builder is able to build the entire product, it should hopefully be able to run your script as well.

Reply:
I have fixed this already, it works now.